packages/waku/src/lib/builder/serve-aws-lambda.ts

```diff
@@ -15,4 +14,32 @@ const app = new Hono();
 app.use('*', serveStatic({ root: `${distDir}/${publicDir}` }));
 app.use('*', honoMiddleware({ loadEntries, ssr, env }));
 
-export const handler = handle(app);
+/*
+ FIX: AWS HTTP API V2 breaks RSC POST requests needed for server actions
+    - AWS HTTP API V2 sets rawPath to a decoded path
+    - We need to encode the string after /RSC/ and replace the orginal rawPath
+    - Hono bug: https://github.com/honojs/hono/issues/2156
+*/
+
+const fixHTTPAPIV2 = (event: LambdaEvent) => {
+  if ('version' in event && event?.version === '2.0') {
+    if (
+      'http' in event.requestContext &&
+      event?.requestContext?.http?.method === 'POST' &&
+      'rawPath' in event &&
+      event?.rawPath?.startsWith('/RSC/')
```

Comment:
`/RSC/` is configurable (theoretically, we've never tried.)
Can you make it `config.basePath + config.rscPath`? We need to get them through `import.meta.env.WAKU_CONFIG_*`.

Reply:
I do not need to change the prefix path `/RSC/` - I would need an other transportation - e.g. by custom header to fix this problem. The problem ist the url encoded data after `/RSC/` - this is decoded by the API Gateway and Waku expects it to be encoded as it is the standard for `c.req.url` from the Hono API.

Reply:
In that case, you can remove L30 condition and encode the entire path.